src/sql/src/plan/statement/dml.rs

```diff
@@ -961,7 +961,9 @@ fn plan_copy_to(
         desc,
         to,
         connection: connection.to_owned(),
+        connection_id: conn_id,
         format_params,
+        max_file_size: options.max_file_size.as_bytes(),
```

Comment:
What is this? It's an option a user can pass to specify the target split size? It probably needs to have some minimum. 1mb, 1kb, 10mb? Dunno.

Reply:
Yes, this is user specified. Technically it's hard to adhere to a minimum, because say there are multiple files, the last file might be tiny because it only had that many leftover records. S3 also does not have a minimum object size.
>Individual Amazon S3 objects can range in size from a minimum of 0 bytes to a maximum of 5 TB.

Reply:
Ok! I was concerned about a user possibly inadvertently setting this to `1` and then we are producing enormous numbers of files and the s3 sink gets significantly backed-up because of the latency to create so many small files. I'm not sure if this is a real concern or not.

Reply:
Added a check that it's at least 16MB (Chose 16MB, because Snowflake defaults to MAX_FILE_SIZE to 16MB)